dotCMS/src/main/java/com/dotmarketing/portlets/htmlpageasset/business/render/HTMLPageAssetRenderedAPIImpl.java

```diff
@@ -342,25 +367,33 @@ private IHTMLPage getPageByUri(final PageMode mode, final Host host, final Strin
         return htmlPage;
     }
 
+    private IHTMLPage getPageById(final PageMode mode, final Host host, final String id)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused method parameters such as 'host'.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4842460)

Reply:
if it not needed, it is a private method, feel free to change the signature to remove it